site/sitemap.xml

```diff
@@ -876,27 +876,27 @@
   <changefreq>weekly</changefreq>
 </url>
 <url>
-  <loc>https://spark.apache.org/screencasts/</loc>
+  <loc>https://spark.apache.org/graphx/</loc>
```

Comment:
You could revert this change, but I'm not sure which version is 'right' - which one is the result as of the latest site generation tools. Rest is OK.

Reply:
Yes, +1 for reverting this file change.

Reply:
Yea, let me revert it.